sklearn/ensemble/_hist_gradient_boosting/_binning.pyx

```diff
@@ -25,21 +28,27 @@ cpdef _map_to_bins(const X_DTYPE_C [:, :] data, list binning_thresholds,
     binning_thresholds : list of arrays
         For each feature, stores the increasing numeric values that are
         used to separate the bins.
+    support_missing_values: ndarray, shape (n_features,)
+        For each feature, indicate whether the first bin is reserved for
+        missing values.
     binned : ndarray, shape (n_samples, n_features)
         Output array, must be fortran aligned.
     """
     cdef:
         int feature_idx
 
     for feature_idx in range(data.shape[1]):
+
         _map_num_col_to_bins(data[:, feature_idx],
                              binning_thresholds[feature_idx],
+                             support_missing_values[feature_idx],
                              binned[:, feature_idx])
 
 
 cdef void _map_num_col_to_bins(const X_DTYPE_C [:] data,
-                               const X_DTYPE_C [:] binning_thresholds,
-                               X_BINNED_DTYPE_C [:] binned):
+                                const X_DTYPE_C [:] binning_thresholds,
+                                const unsigned char support_missing_values,
```

Comment:
nit: now regardless of the training data including a missing value for a feature or not, we do _support_ missing values. This flag is more like "includes_missing_values" rather than "support_missing_values". Is this a correct understanding?

Reply:
`includes_missing_values` would suggest that there are missing values in `data`, but as you noted that's not the purpose of this parameter.